jobs/build-arch.Jenkinsfile

```
@@ -463,18 +461,18 @@ lock(resource: "build-${params.STREAM}-${params.ARCH}") {
             // see https://github.com/coreos/fedora-coreos-tracker/issues/920#issuecomment-914334988
             // Right now we only patch if platforms.yaml hasn't made it to this stream yet.
             // Fold this back into the above parallel runs (i.e. add to config.yaml
-            // artifacts list for aarch64 and delete below code) once platforms.yaml
+            // artifacts list for aarch64 and delete below code and knob) once platforms.yaml
             // exists everywhere. https://github.com/coreos/fedora-coreos-config/pull/1181
             if (basearch == "aarch64") {
                 stage('AWS') {
-                    shwrap("""
-                    if [ "${pipetype}" == "fcos" ]; then
+                    if (pipecfg.aarch64_serial_console_hack) {
```

once you move the vaule in the config I think it will be:

```suggestion
                    if (pipecfg.aws_aarch64_serial_console_hack) {
```